apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -401,6 +408,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.Input.prototype.setStrictCheck = function (check) {
     return this.setCheck(check);
   };
+  blocklyWrapper.Block.prototype.setStrictOutput = function (isOutput, check) {
```

Comment:
Could you explain why didn't we need to define `setStrictCheck` in Google Blockly before this PR?

Reply:
Thanks! I meant to ask why `setStrictOutput` didn't need to be added before this PR but luckily you answered the question I intended - thanks!
So did the 3 Sprite Lab blocks not exist beforehand?

Reply:
Discussed offline. Adding `setStrictOutput` is in anticipation of moving Sprite lab (including the 3 blocks that reference `setStrictOutput`) to mainline. Thanks @mikeharv!